main.js

```diff
@@ -337,14 +338,18 @@ function _isObject({credential}) {
  * @param {object} options - Options to use.
  * @param {object} options.credential - A VC with a credentialStatus.
  *
- * @returns {Array<object>} An array of statuses.
+ * @returns {Array<object>} An array of statuses with type
+ *   "StatusList2021Entry".
  */
 function _getStatuses({credential}) {
   const {credentialStatus} = credential;
-  if(Array.isArray(credentialStatus)) {
-    return credentialStatus;
+  if(Array.isArray(credentialStatus) && credentialStatus.length > 0) {
+    return credentialStatus.filter(cs => cs.type === 'StatusList2021Entry');
```

Comment:
This can return an empty array -- and then the checks above that are supposed to throw when there are no matching types won't throw. Looks like we need another test with an empty `credentialStatus` array on a VC.

Reply:
Fixed and added tests here [53a86b2](https://github.com/digitalbazaar/vc-status-list/pull/6/commits/53a86b20e3e57c87d9239f81d97953a9897b39e3)